src/appmixer/akamai/clientlist/UpsertListEntries/component.json

```diff
@@ -0,0 +1,178 @@
+                "upsert": { "type": "object" }
+            },
+            "required": [
+                "listId"
+            ]
+        },
+        "inspector": {
+            "inputs": {
+                "listId": {
+                    "type": "text",
+                    "label": "List",
+                    "index": 0,
+                    "source": {
+                        "url": "/component/appmixer/akamai/clientlist/GetLists?outPort=out",
+                        "data": {
+                            "transform": "./GetLists#listsToSelectArray",
+                            "messages": {
+                                "in/isSource": true
+                            }
+                        }
+                    }
+                },
+                "upsert": {
+                    "type": "expression",
+                    "label": "Append or Update",
+                    "index": 1,
+                    "levels": ["ADD"],
+                    "fields": {
+                        "value": {
+                            "type": "text",
+                            "label": "Value",
+                            "required": true,
+                            "tooltip": "Value of the entry, which is either an IP address, an Autonomous System Number (ASN), a Geolocation, a TLS fingerprint, or a file hash.",
+                            "source": {
+                                "url": "/component/appmixer/akamai/clientlist/GetListEntries?outPort=out",
+                                "data": {
+                                    "transform": "./GetListEntries#listEntriesToSelectArray",
+                                    "messages": {
+                                        "in/listId": "inputs/in/listId",
+                                        "in/isSource": true
+                                    }
+                                }
+                            }
+                        },
+                        "description": {
+                            "type": "text",
+                            "label": "Description",
+                            "tooltip": "Any additional description of the client list entry."
+                        },
+                        "expirationDate": {
+                            "type": "number",
+                            "label": "TTL",
+                            "tooltip": "Time to live in seconds indicating when the entry expires."
+                        },
+                        "tags": {
+                            "type": "text",
+                            "label": "Tags",
+                            "tooltip": "Any defined tags for a specified client list. Separate by a comma. Example: <code>tag1,tag2,tag3</code>. In case of update, the tags will be added to existing ones."
+                        }
+                    }
+                }
+            }
+        }
+    }
+  ],
+  "outPorts": [
+    {
+        "name": "out",
+        "options": [
```

Comment:
please do not return 
```
appended: [...],
updated:[..]
```
instead return array of updated / added items, with additional property "action". action will be "updated" or "added". 

```
[
   { "action": "updated", "value": 1.1.1.1 ... },
   { "action": "added", "value": 1.1.1.2}
]
```